test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -19,12 +19,14 @@
 #include "backend/protobuf/transaction.hpp"
 #include "builders/protobuf/queries.hpp"
 #include "builders/protobuf/transaction.hpp"
+#include "cryptography/crypto_provider/crypto_signer_impl.hpp"
 #include "cryptography/crypto_provider/crypto_defaults.hpp"
 #include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
 #include "datetime/time.hpp"
 #include "framework/integration_framework/integration_test_framework.hpp"
 #include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
 #include "model/generators/query_generator.hpp"
+#include "model/sha3_hash.hpp"
```

Comment:
sha3 looks doubtful here. We have to use shared model here.

Reply:
There is still old model that uses hash(). Reworking it with shared_model is not related to the PR.